src/Syntax/NativeSyntaxHighlights.re

```diff
@@ -33,22 +33,6 @@ type t =
     })
     : t;
 
-let _hasTreeSitterScope = (configuration, scope: string) => {
-  let treeSitterEnabled =
-    Core.Configuration.getValue(c => c.experimentalTreeSitter, configuration);
-
-  if (!treeSitterEnabled) {
-    false;
-  } else {
-    switch (scope) {
-    | "source.json" => true
-    /*  | "source.c" => true
-        | "source.cpp" => true */
-    | _ => false
-    };
-  };
-};
-
```

Comment:
Dropped since I don't think we need this and a `getParser` (or something similar), so we may as well just do the config look up inline. The config lookup in the current version is actually duplicated for some reason.

Reply:
Ah, ya, I just saw this too (working on #1626). Seems reasonable to me, and should be easy to merge with #1626 👍